### DISCUSSION TITLE
How to send data to a new connection?

### DISCUSSION THREAD

**Discussion by ilayya** [beyondcode/laravel-websockets#806] [Q&A]
My task is to display up-to-date data, they can be updated either every second or every few minutes. How can I give him the necessary data when connecting a new user?

**Comment by cbrhex** [comment]
Seems like this project is dead =(

**Comment by amir-khoshbakht** [comment]
This will be a bit technical. To do this, you need to know how [Ratchet ](https://github.com/ratchetphp/Ratchet) and [reactphp](https://github.com/reactphp/reactphp) work.
Description : 
https://beyondco.de/docs/laravel-websockets/advanced-usage/custom-websocket-handlers
https://beyondco.de/docs/laravel-websockets/advanced-usage/app-providers
